src/app/file-manager/file-explorer-upload/file-explorer-upload.component.ts

```diff
@@ -231,7 +229,10 @@ export class FileExplorerUploadComponent implements OnInit {
               progress: 0,
               status: ItemStatus.PENDING,
               transferred: 0,
-              remaining: 0
+              remaining: 0,
+              bucketName: '',
+              delimiter: '',
+              prefix: ''
```

Comment:
Instead of duplicating the default settings for these fields, can you add a constructor to `Item` that handles this?